src/components/TheTodoList.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <!-- todo items -->
+  <ul class="todo-items">
+    <TheTodoListItem
+      v-for="(todo, index) in todos"
+      :key="todo.id"
+      :itemIndex="index"
+      :onDone="onDone"
+      :onDelete="onDelete"
+      :onEdit="onEdit"
+      v-bind="todo"
+    />
+  </ul>
+</template>
+
+<script>
+import TheTodoListItem from './TheTodoListItem'
+
+export default {
+  name: 'TodoItems',
+
+  components: {
+    TheTodoListItem,
+  },
+
+  data: () => ({
+    time: 'descending',
+  }),
+
+  props: {
+    todos: Array,
```

Comment:
Write complete details for each props like the instruction:

// Even better!
props: {
status: {
type: String,
required: true,
validator: function (value) {
return [
'syncing',
'synced',
'version-conflict',
'error'
].indexOf(value) !== -1
}
}
}